spec-tests/src/test/java/org/candlepin/spec/jobs/InactiveConsumerCleanerJobSpecTest.java

```diff
@@ -114,8 +115,9 @@ public void shouldDeleteInactiveConsumers() throws Exception {
         final String inactiveConsumer1Uuid = inactiveConsumer.getUuid();
         assertGone(() -> consumerApi.getConsumer(inactiveConsumer1Uuid));
         // Verify that the inactive consumer has been moved to the deleted_consumers table.
+        DateTimeFormatter fmt = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
         List<DeletedConsumerDTO> deletedConsumers = deletedConsumerApi
-            .listByDate(inactiveConsumer.getCreated().toString());
+            .listByDate(fmt.format(inactiveConsumer.getCreated()));
```

Comment:
I think we should be able to use the the ISO-8601 static formatters that DateTimeFormatter already has instead of creating a new one. I believe that the ISO_DATE_TIME formatter should work since it includes the offset data that is included in the OffsetDateTime object that is returned from `inactiveConsumer.getCreated()`.

`DateTimeFormatter.ISO_DATE_TIME.format(inactiveConsumer.getCreated());`